Assets/Character/AaronTestAssets/Player.cs

```diff
@@ -0,0 +1,60 @@
+﻿using UnityEngine;
+using System.Collections;
+
+
+
+public class Player : MonoBehaviour {
+
+
+    public float speed = 2f;
+    private Animator animator;
+    private Rigidbody2D rigidbody;
+	// Use this for initialization
+	void Start () {
+
+        animator = GetComponent<Animator>();
+        rigidbody = GetComponent<Rigidbody2D>();
+       // if (animator.)
+	
+	}
+	
+	// Update is called once per frame
+	void Update () {
+        int horizontal = 0;
+        
+
+        horizontal = (int)(Input.GetAxisRaw("Horizontal"));
+
+        transform.position += new Vector3(horizontal, 0, 0) * speed * Time.deltaTime;
+        if (horizontal > 0 && animator.GetBool("OnGround"))
```

Comment:
if (animator.getBool("OnGround"))
{
    if horiz is pos ...
    else if horiz is neg ...
    if spacebar down ...
}

Makes if statements less redundant with OnGround